### DISCUSSION TITLE
Slack

### DISCUSSION THREAD

**Discussion by bdsoha** [skyvar/.github#7] [Announcements]
@skyvar/everyone 
Hello,

All _SkyVar_ members should have received an invitation to _Slack_.
In the next day or two, I will post the following in regards to using slack:
- Conventions.
- Limitation.
- Best practices.

In the meantime, please:
- Install it.
- Accept the invitation .
- Update your profile to match your GitHub profile.
- Add your telephone contact information to the profile in the form of `+972-5*-***-****`.

**Comment by FridaPshitik** [comment]
The `Slack` installations
and the confirmation sent by email are blocked from `netfree`

**Reply by bdsoha** [reply]
This should have been unblocked already.
Please contact one of the administrators.

**Reply by EsterWeiss** [reply]
The blockage has been resolved.

**Comment by bdsoha** [comment]
# Overview *(short)*

## Terms

- **Channel:** Channels are spaces for all the people, tools, and files you need to get work done in Slack. You can create channels for different teams, topics, and projects to bring order and clarity to work. 
- **DM:** Direct messages are one-to-one conversations between you and another member.
- **Display name:** A person's first name, or nickname listed in their profile. This is what people would like to be referred to in Slack.
- **Do Not Disturb *(DND)*:** Setting Do Not Disturb hours will automatically disable notifications while you’re not working, whereas manually pausing notifications allows you to disable them on the fly. When DND or snooze is in effect, Slack will not send you notifications for the messages or `@mentions` you receive.
- **Group direct message *(Group DM)*:** Group direct messages are smaller discussions, outside of channels, between you and up to eight other members.
- **Huddles:** A voice conversation in a Slack channel or direct message that can include video and screensharing.
- **Post:** Posts let you share long-form, formatted content — like project plans or documentation — directly in Slack.
- **Reminders:** Send yourself or other members a reminder for anything: important meetings, to-do items, and even motivational messages.
- **Slash commands:** Slash commands act as shortcuts that trigger messages or actions from Slack, an app, or an integration. 

## Conventions

- A channel can be public *(with a #️⃣ prefixing the name)* or private *(with a 🔒 prefixing the name)*. We believe in transparency; therefore, only dedicated project channels will be marked as private. Be considerate when interacting with channels you don’t directly belong to.
- Some channels are prefixed to indicate their use-case:
    - `#help-***`: is meant for help regarding a specific topic. For example, `#help-tech-it` is to be used when IT assistance is required.
    - `#team-***`: is meant for members that have been allocated as field experts.  For example, `#team-devops` is for member to discuss issues, features, development, etc. in the realm of DevOps.
    - `#site-***`: is meant for physical office locations. Office related matters or announcements can be posted here.
    - `#project-***`: is meant for project development purposes.

## Rules

- Slack is a popular workplace communication app used across multiple platforms and devices by individuals, teams, and companies.
- It offers messaging functionality, sharing files and media, initiating voice and video calls, project planning, and integrating tools and add-ons to help projects flow seamlessly.
- It is no means a **chatroom**. Therefore, don't be too casual. Slack is a workplace tool and not social media. Keep it professional always.
- Slack has the ability to share files between users or channels, but is not a *file storage* system. Also, files can and will be removed after a certain amount of time *(see limitations)*.
- Do not open unnecessary DMs or group DMs, use channels instead.
- Even though you are authorized to contribute to public channels, be considerate and don’t post unless you are a member *(or required to do so)*.

## Limitations

- Message history will be *cyclically* removed after 90-days or 10,000 messages. Therefore, do not rely on Slack for historical data. If chat content is considered important, save it elsewhere.
- Huddles can only be 1-on-1.
